DEPS

```diff
@@ -16,7 +16,7 @@ deps = {
   "vendor/bip39wally-core-native": "https://github.com/brave-intl/bip39wally-core-native.git@9b119931c702d55be994117eb505d56310720b1d",
   "vendor/bat-native-anonize": "https://github.com/brave-intl/bat-native-anonize.git@e3742ba3e8942eea9e4755d91532491871bd3116",
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@1b4362968c8f22720bfb75af6f506d4ecc0f3116",
-  "components/brave_sync/extension/brave-sync": "https://github.com/brave/sync.git@896d733ddcf5fb1a99611a0830d4824b4cb22a68",
+  "components/brave_sync/extension/brave-sync": "https://github.com/brave/sync.git@951e07120166ab1d7b1857424676c6ae1b27f49f",
```

Comment:
we need new deps once https://github.com/brave/sync/pull/305 merged and cherry-picked into master

Reply:
done
https://github.com/brave/brave-core/pull/2010/commits/dec70a71da102678f1a36bee5b708b85c2f62b59